### PR TITLE
Add page templates

### LINKED ISSUE #125: Add templates
We need to add a number of different templates, and set the _Wide Width, Title_ template as the default for courses.

- Wide Width, No Title (this is already available [here](https://github.com/Automattic/course/blob/trunk/templates/page-wide-width.html))
- Wide Width, Title
- Single Column, No Title
- Single Column, Title

#### Scenario: New Course
**Given** The Course theme is activated
**When** I create a new course
**And** select any course pattern
**Then** the _Template_ is set to _Wide Width, Title_
**And** the course title is displayed and the page is set to wide width on the frontend.

#### Scenario: Existing Course
**Given** The Course theme is activated
**When** I edit an existing course
**Then** the _Template_ is set to _Wide Width, Title_
**And** the course title is displayed and the page is set to wide width on the frontend.

#### Scenario: Course Templates
**Given** The Course theme is activated
**When** I edit an existing course
**And** I click on the _Template_ dropdown
**Then** I am able to switch between the different templates
**And** the course updates accordingly on the frontend.

#### Scenario: Page Templates
**Given** The Course theme is activated
**When** I edit an existing page
**And** I click on the _Template_ dropdown
**Then** I am able to switch between the different templates
**And** the page updates accordingly on the frontend.

### PR DESCRIPTION
**Pull request by gabrielcaires**
Fix #125 
Scope updated following this thread  p1669830788095469/1669826875.239069-slack-C044ERD5WDV

### Introduced Changes
- Add Single Column
- Add Single Column - Title Only
- Add Single Column - Featured Image Only
- Add Wide
- Add Wide - Title Only
- Add Wide - Featured Image Only
- Single Course  == Wide Template

## Single Column
Single Column, Featured Image, Title

## Single Column - Title Only
Single Column, No Featured, Title

### Single Column - Feature Image Only
Single Column, Featured Image, No Title

###  Single Column -  No Featured Image, No Title
Single Column, No Featured Image, No Title

###  Wide
Wide, Featured Image, Title

### Wide - Featured Image Only
Wide, Featured Image, No Title

###  Wide, No Featured Image, No Title
Wide, No Featured Image, No Title

### Wide - Title Only
Wide, No Featured Image, Title

### CODE REVIEW

**Comment by donnapep** [comment]
Did a quick test, but haven't looked at the code yet.

The template dropdown looks like this for pages. Looks like we're missing a proper name for the last one. Is it supposed to be the _Wide Width - Title_ one?

Screenshot 2022-11-30 at 11 00 52 AM

For posts, I see this. Same issue, and looks like one template is missing:

Screenshot 2022-11-30 at 11 03 10 AM

And this is for courses:

Screenshot 2022-11-30 at 11 08 11 AM

When I view an existing course on the frontend, I no longer have any margins and my featured image is gone. It doesn't look like any of the templates are showing my featured image. It's possible there's other content missing as well:

screencapture-senseitheme-local-course-getting-started-with-sensei-lms-2022-11-30-11_21_08

This is what it looks like in trunk:

screencapture-senseitheme-local-course-getting-started-with-sensei-lms-2022-11-30-11_20_42

**Comment by gabrielcaires** [comment]
@donnapep Investigating because the correct name is set here https://github.com/Automattic/course/pull/126/files#diff-160c735a986cf8c05e1f6b2e9ac89dd0410562d2fe5ccbdc93157bd92f3dc2bcR15
And it working for me 🤔 

Screenshot 2022-11-30 at 13 22 11

**Comment by gabrielcaires** [comment]
@donnapep PR updated with the new templates.

File: theme.json [hunk 1]
```diff
@@ -6,13 +6,82 @@
...
+			"title": "Wide - Title Only",
+			"postTypes": [
+				"page",
+				"post",
+				"course"
+			]
+		},
+		{
+			"name": "wide-featured-no-title",
+			"title": "Wide - Feature Image Only",
```
Comment (donnapep) [inline theme.json]:
```suggestion
			"title": "Wide - Featured Image Only",
```

Comment (gabrielcaires) [reply theme.json]: Fixed here [9b6b19a](https://github.com/Automattic/course/pull/126/commits/9b6b19acc60de78287d9178eaf644a01d6c75195)

File: templates/archive.html [hunk 2]
```diff
@@ -34,4 +34,4 @@
 </div>
 <!-- /wp:group -->
 
-<!-- wp:template-part {"slug":"footer","theme":"course","tagName":"footer"} /-->
```
Comment (gabrielcaires) [inline templates/archive.html:37]: I remove this because it is breaking on my machine.
We can have another Pr only to add it on all files.

**Comment by donnapep** [comment]
After the merge I'm seeing `page-wide-width` in the template selector. It wasn't showing that until I pulled the merged changes:

Screenshot 2022-12-02 at 12 20 54 PM

**Comment by donnapep** [comment]
Oh, maybe you just fixed it in your latest commit. 😆 

Yup, all good!

**Comment by gabrielcaires** [comment]
> Oh, maybe you just fixed it in your latest commit. 😆
> 
> Yup, all good!

Yes :D

**Review by donnapep — APPROVED** [review]

**Pull request merged**